tests/roles/run_tests/vars/main.yml

```diff
@@ -87,6 +87,10 @@ IRONIC_TLS_SETUP: "{{ lookup('env', 'IRONIC_TLS_SETUP') | default('true', true)
 IRONIC_BASIC_AUTH: "{{ lookup('env', 'IRONIC_BASIC_AUTH') | default('true', true) }}"
 IRONIC_USE_MARIADB: "{{ lookup('env', 'IRONIC_USE_MARIADB') | default('false', true) }}"
 
+# Metrics
+CAPM3_DIAGNOSTICS_ADDRESS: "localhost:8080"
```

Comment:
Are these related to CAPI bump (other than helping debug things?)

Reply:
Related to CAPI BUMP yes, CAPI has enabled this from CAPI v1.6 and we are following up with the same in CAPM3.